ROX-19799: Update docs for VM2.0 (#78059)
openshift/openshift-docs: operating/manage-vulnerabilities/vulnerability-management-dashboard.adoc

:_mod-docs-content-type: ASSEMBLY
[id="vulnerability-management-dashboard"]
= Using the vulnerability management dashboard (deprecated)
include::modules/common-attributes.adoc[]
:context: vulnerability-management-dashboard

toc::[]

[role="_abstract"]

> Historically, {product-title-short} provided a view of vulnerabilities discovered in your system in the vulnerability management dashboard. With the dashboard, you can view vulnerabilities by image, node, or platform. You can further view vulnerabilities by clusters, namespaces, deployments, node components, and image components. The dashboard is deprecated in {product-title-short} 4.5 and will be removed in a future release.

Reviewer: ```suggestion
Historically, {product-title-short} has provided a view of vulnerabilities discovered in your system in the vulnerability management dashboard. The dashboard enables you to view vulnerabilities by image, node or platform. You can also view vulnerabilities by clusters, namespaces, deployments, node components and image components. The dashboard is deprecated in {product-title-short} 4.5 and will be removed in a future release.
```

> [NOTE]

Reviewer: ```suggestion
[IMPORTANT]
```
